base_rest_pydantic/restapi.py

```diff
@@ -154,10 +159,23 @@ def from_params(self, service, params):
 
     def to_response(self, service, result):
         self._do_validate(result, "output")
-        return [
+        json_res = {}
+        json_res["metadata"] = self.to_metadata_response(service, result["metadata"])
+        json_res["data"] = [
             super(PydanticModelList, self).to_response(service=service, result=r)
```

Comment:
I dislike when we super() and change the object hierarchy.

Can we do something like:
```suggestion
            self.to_data_response(service=service, result=r)
```